packages/jest-expect-har/test/tsconfig.json

```diff
@@ -1,9 +1,7 @@
 {
   "extends": "../tsconfig.json",
   "compilerOptions": {
-    "module": "es6",
-    "moduleResolution": "node",
-    "strict": false
+    "resolveJsonModule": true
```

Comment:
huh kinda surprising this isn’t in our base config

Reply:
we only ever load JSON files in tests